test/config_test.py

```diff
@@ -13,14 +13,16 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """Tests for config_loader.config."""
+import abc
 import contextlib
 import tempfile
 
 from absl.testing import absltest
-from config_loader import config
 import jinja2
 import yaml
 
+from yaml_config_loader import config
+
 
 @contextlib.contextmanager
 def config_file(content):
```

Comment:
I know it's not related to this PR, but I like this helper. I'm going to steal this for other stuff. :)